tensorflow/training/docker/2.5/py3/Dockerfile.cpu

```diff
@@ -136,7 +136,9 @@ RUN ln -s /etc/ssl/certs/*.* /usr/local/ssl/certs/
 RUN ln -s $(which ${PYTHON}) /usr/local/bin/python \
  && ln -s $(which ${PIP}) /usr/bin/pip
 
-RUN apt-get update && apt-get -y install protobuf-compiler
+RUN apt-get update \
+    && apt-get -y upgrade \
```

Comment:
Please move this to the apt-get update/install `RUN` statement at the start of the dockerfile.

Also, please add a
```
 && rm -rf /var/lib/apt/lists/* \
 && apt-get clean
```
at the end of this `RUN` statement.